# fontconfig complains with the config file when installing the font:
# "Having multiple values in <test> isn't supported and may not work as expected"
# This is a fix for that error.
--- a/44-wqy-zenhei.conf	2009-05-17 14:25:41
+++ b/44-wqy-zenhei.conf	2024-01-30 06:36:37
@@ -1,46 +1,29 @@
 <?xml version="1.0"?>
 <!DOCTYPE fontconfig SYSTEM "fonts.dtd">
-<!-- WenQuanYi Zen Hei Configure File -->
 <fontconfig>
-  <!-- modified by Madsen@SMTH -->
-  <match target="font">
-        <test qual="any" name="family">
-                <string>WenQuanYi Zen Hei</string>
-                <string>文泉驿正黑</string>
-                <string>文泉驛正黑</string>
-        </test> 
-        <edit name="globaladvance"><bool>false</bool></edit>
-        <edit name="spacing"><int>0</int></edit>
-        <edit name="antialias" mode="assign"><bool>true</bool></edit>
-        <edit name="hinting" mode="assign"><bool>true</bool></edit>
-        <edit name="hintstyle" mode="assign"><const>hintnone</const></edit>
-        <edit name="autohint" mode="assign"><bool>false</bool></edit>
-        <edit name="rh_prefer_bitmaps" mode="assign"><bool>false</bool></edit>
-        <edit name="rgba" mode="assign"><const>none</const></edit>
-        <edit name="embeddedbitmap"><bool>false</bool></edit>
-  </match>
-        <alias>
-                <family>serif</family>
-                <prefer>
-                        <family>Bitstream Vera Serif</family>
-                        <family>DejaVu Serif</family>
-                        <family>WenQuanYi Zen Hei</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>sans-serif</family>
-                <prefer>
-                        <family>DejaVu Sans</family>
-                        <family>Bitstream Vera Sans</family>
-                        <family>WenQuanYi Zen Hei</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>monospace</family>
-                <prefer>
-                        <family>DejaVu Sans Mono</family>
-                        <family>Bitstream Vera Sans Mono</family>
-                        <family>WenQuanYi Zen Hei Mono</family>
-                </prefer>
-        </alias>
+	<alias>
+		<family>serif</family>
+		<prefer>
+			<family>Bitstream Vera Serif</family>
+			<family>DejaVu Serif</family>
+			<family>WenQuanYi Zen Hei</family>
+		</prefer>
+	</alias>
+	<alias>
+		<family>sans-serif</family>
+		<prefer>
+			<family>DejaVu Sans</family>
+			<family>Bitstream Vera Sans</family>
+			<family>WenQuanYi Zen Hei</family>
+		</prefer>
+	</alias>
+	<alias>
+		<family>monospace</family>
+		<default>WenQuanYi Zen Hei Mono</default>
+		<prefer>
+			<family>DejaVu Sans Mono</family>
+			<family>Bitstream Vera Sans Mono</family>
+			<family>WenQuanYi Zen Hei Mono</family>
+		</prefer>
+	</alias>
 </fontconfig>
